### PR TITLE
Adding a full automation service call example

### PR DESCRIPTION
**Pull request by kozoke**
I did not realize I could add additional configurations by adding a second 'data' section.  May just be my ignorane but thought we could add a full example to help anyone else.

## Proposed change

Simple change to demonstrate the addtion of the second 'data' section to leverage additional android features.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ X ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @kozoke, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20921).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/nfandroidtv.markdown [hunk 1]
```diff
@@ -125,4 +125,20 @@ icon:
   auth: "digest"
 ```
 
+Example of automation serivce call, full cofiguration:
+
+```yaml
+  - service: notify.living_room_tv
+    data:
+      title: "Thanks for the water!"
+      message: "Nigel is {{ states.sensor.nigel_moisture.state }}% moisture"
```
Comment (frenck) [inline source/_integrations/nfandroidtv.markdown]: Don't access `states` directly, as it won't be able to handle occurrences when the entity isn't available/loaded yet.

Instead use the `states()` function

File: source/_integrations/nfandroidtv.markdown [hunk 2]
```diff
@@ -125,4 +125,20 @@ icon:
...
+      data:
+        duration: 4
+        position: "bottom-left"
+        fontsize: "medium"
+        transparency: "80%"
+        color: "teal"
+        interrupt: 0
+ ```yaml
```
Comment (frenck) [inline source/_integrations/nfandroidtv.markdown]: You can remove `yaml` here

File: source/_integrations/nfandroidtv.markdown [hunk 3]
```diff
@@ -125,4 +125,20 @@ icon:
   auth: "digest"
 ```
 
+Example of automation serivce call, full cofiguration:
```
Comment (frenck) [inline source/_integrations/nfandroidtv.markdown]:
```suggestion
Example of an automation with an service call, full configuration:
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/nfandroidtv.markdown (hunk 1 above)
Comment (kozoke) [reply source/_integrations/nfandroidtv.markdown]: Understood.  Not quite sure how to edit it but I will try or resubmit a PR if it does not work.  (Still learning) :-)

Comment (kozoke) [inline source/_integrations/nfandroidtv.markdown]: - service: notify.living_room_tv
      data:
        title: "Thanks for the water!"
        message: Nigel is {{ states('sensor.nigel_moisture') }}% moisture
        data:
          duration: 4
          position: "bottom-left"
          fontsize: "medium"
          transparency: "80%"
          color: "teal"
          interrupt: 0

File: source/_integrations/nfandroidtv.markdown (hunk 2 above)
Comment (kozoke) [reply source/_integrations/nfandroidtv.markdown]: I'm not sure how this works 'you can remove yaml here' - I am clicking on th etext but I can't edit ithere.  I'll go back to the PR and try.  Thanks again for your patience.

**Review by kozoke — COMMENTED** [review]
Sorry Frenck, I cannot figure out how to make the change.  Do I cancel this pull request and make a new committ and pull request?

File: source/_integrations/nfandroidtv.markdown (hunk 1 above)
Comment (kozoke) [reply source/_integrations/nfandroidtv.markdown]: Hi Frenck.  Big fan of your work.   I'm a bit new to using git etc and appreciate your patience.  
I agree with the changes/suggestions. I'm not sure how to implment the states() change - I have it fixed but unsure if I need cancel this PR (unlikely) or somehow edit this one?  I'll do some git research to address it.  thx.

**Comment by kozoke** [comment]
Hi. I made the changes in my patch-1 branch and committed the change.  I'm not sure how to change this PR to include that change?  :-)

File: source/_integrations/nfandroidtv.markdown [hunk 4]
```diff
@@ -125,4 +125,20 @@ icon:
...
+      title: "Thanks for the water!"
+      message: Nigel is {{ states('sensor.nigel_moisture') }}% moisture
+      data:
+        duration: 4
+        position: "bottom-left"
+        fontsize: "medium"
+        transparency: "80%"
+        color: "teal"
+        interrupt: 0
+ ```yaml
```
Comment (frenck) [inline source/_integrations/nfandroidtv.markdown]:
````suggestion
{% raw %}

```yaml
service: notify.living_room_tv
data:
  title: "Thanks for the water!"
  message: "Nigel is {{ states('sensor.nigel_moisture') }}% moisture"
  data:
    duration: 4
    position: "bottom-left"
    fontsize: "medium"
    transparency: "80%"
    color: "teal"
    interrupt: 0
```

{% endraw %}
````

**Review by frenck — APPROVED** [review]
Thanks, @kozoke 👍

**Pull request merged**